widget/table.go

```diff
@@ -90,41 +79,52 @@ func (t *Table) CreateRenderer() fyne.WidgetRenderer {
 	return r
 }
 
-// Selection will mark the specified cell (at row, col) to be marked as selected.
-func (t *Table) Selection() (row, col int) {
-	return t.selectedRow, t.selectedColumn
+// Select will mark the specified cell as selected.
+func (t *Table) Select(id *CellID) {
+	if id == nil {
+		return
+	}
+	t.Unselect(t.selectedCell)
```

Comment:
This will result in `moveCallback` being called twice

Reply:
Also it should not be called if the `selectedCell` is nil

Reply:
Resolved